### ISSUE TITLE
ErrorException after updating to 2.10.3

### ISSUE THREAD

**Issue by u12206050** [beyondcode/tinkerwell-community#344]
Everything worked until I updated today.

When ever I try to load a class I get the following error output not matter which class it is:

```

   ErrorException 

  include(phar:///Applications/Tinkerwell.app/Contents/Resources/tinkerwell/tinker.phar/vendor/composer/../composer/InstalledVersions.php): failed to open stream: phar error: "vendor/composer/InstalledVersions.php" is not a file in phar "/Applications/Tinkerwell.app/Contents/Resources/tinkerwell/tinker.phar"

  at phar:///Applications/Tinkerwell.app/Contents/Resources/tinkerwell/tinker.phar/vendor/composer/ClassLoader.php:444
    440▕  * Prevents access to $this/self from included files.
    441▕  */
    442▕ function includeFile($file)
    443▕ {
  ➜ 444▕     include $file;
    445▕ }
    446▕

      [2m+2 vendor frames [22m
  3   [internal]:0
      Composer\Autoload\ClassLoader::loadClass("PackageVersions\Versions")

      [2m+24 vendor frames [22m
  28  [internal]:0
      Illuminate\Foundation\Application::Illuminate\Foundation\{closure}(Object(Sentry\Laravel\ServiceProvider))
```

**Comment by albalooshi** [comment]
+1

**Comment by shoemoney** [comment]
At first I was sure it was the upgrade but then noticed homebrew updated to php8.   I go to 7.4 and its fine.

**Comment by njames** [comment]
+1 I updated to 2.10.3 and got this issue ... dropped back to 2.10.2 and all fine.

**Comment by ReckeDJ** [comment]
+1 Same issue here, works fine on 2.10.2. But, auto-update updates to 2.10.3 after reopening Tinkerwell.

**Comment by sschlein** [comment]
Fixed with 2.11 a few moments ago - sorry. Never deploy on a Friday, right.
